run_VAE.py

```diff
@@ -1,8 +1,11 @@
 from pipeline.patch_VAE import assemble_VAE, process_VAE, trajectory_matching
-from torch.multiprocessing import Pool, Queue, Process
+# from torch.multiprocessing import Pool, Queue, Process
 import torch.multiprocessing as mp
-import os
+import os, sys
 import argparse
+from multiprocessing import Process
```

Comment:
hm, i'm not sure i intended to change this in this PR.  This might reflect and older version of the script that hadn't been updated.  Will change it back.